react/server.js

```diff
@@ -0,0 +1,256 @@
+// Express server previewing how to best interact with common Pelm API endpoints
+import express from 'express';
+import fetch, { Headers } from 'node-fetch';
+import dotenv from 'dotenv';
+import session from 'express-session'
+const app = express()
+const port = 3001
+dotenv.config()
+
+// Enviornment variables stored in the .env file you created
+const PELM_CLIENT_ID = process.env.PELM_CLIENT_ID
+const PELM_SECRET = process.env.PELM_SECRET
+const USER_ID = process.env.USER_ID
+
+
+app.use(
+  // Here we are using session in place of a database
+  // Instead use your own database to store values like the Authorization_Token
+  session({ secret: "secret", saveUninitialized: true, resave: true })
+);
+
+
+app.use(express.json())
+app.use(express.urlencoded({ extended: true }))
+
+
+// Create connect_token
+// Link to Pelm docs: https://docs.pelm.com/reference/post_auth-connect-token
+app.get('/connect', (req, res) => {
+
+  const headers = new Headers();
+  headers.set('Pelm-Client-Id', PELM_CLIENT_ID);
+  headers.set('Pelm-Secret', PELM_SECRET);
+
+  const encodedParams = new URLSearchParams();
+
+  encodedParams.set('user_id', USER_ID);
+
+  const requestOptions = {
+    method: 'POST',
+    headers,
+    body: encodedParams,
+  };
+
+  // Reach out to the Pelm API connect endpoint providing all the relevent headers
+  fetch('https://api.pelm.com/auth/connect-token', requestOptions)
+    .then(response => {
+      if (response.ok) {
+        return response.json();
+      } else {
+        return response.text().then(text => { throw new Error(text) })
+      }
+    })
+    .then((data) => {
+      // Return connect token
+      res.end(JSON.stringify({
+        connectToken: data['connect_token']
+      }))
+    })
+    .catch((error) => {
+      try {
+        const errorObject = JSON.parse(error.message);
+        console.log(errorObject)
+      } catch (e) {
+        console.log("an error occurred")
+      }
+    });
+})
```

Comment:
Can we actually return the entire response to the frontend so that the frontend can display errors if they occur.

And the frontend can handle parsing the response.

something like:
```
.then(response => {
return response.json()
}
```

^ I think that works? You might need to play around with it

Reply:
Same thing for all other endpoints except `/authorization`

Reply:
Added in both react and vanilla_js implementations for /connect-token and /authorization